guide/language-concepts/bodies.markdown

```diff
@@ -55,6 +55,57 @@ The attributes within the body are then type specific. Bodies of type `perms` co
 
 Such bodies can be reused in multiple promises. Like bundles, bodies can have parameters. The body `mog` also consists of the file permissions, file owner, and file group, but the values of those attributes are passed in as parameters.
 
+#### Body Inheritance
+
+CFEngine 3.8 introduced body inheritance via the `inherit_from`
+attribute. It's a parameterized single-inheritance system, so a body can only inherit from one other body, and it can apply parameters.
+
+Let's see it with the `system` and `mog` bodies from earlier:
+
+```cf3
+    body perms system
+    {
+      mode => "644";
+      owners => { "root" };
+      groups => { "root" };
+    }
+
+    body perms system_inherited
+    {
+      inherit_from => mog("644", "root", "root");

```

Comment:
Latest in the inheritance chain wins.  I didn't think of that in the example, though.  Do you think it's worth including it (keeping in mind it's a beginner document) or should I put it elsewhere?